README.md

```diff
@@ -14,7 +14,11 @@ some of our favorite tools:
 * tabelle
 * Custom Elements
 
-## Setup (without Docker)
+## Setup
+
+[Install Node.js](https://nodejs.org/en/download/)
+
+In this directory run:
 
 ```
 npm i
```

Comment:
```suggestion
npm install
```

Given the context, we wanna avoid unnecessarily obscure instructions.

Reply:
I agree, any further feedback on the README?

Reply:
Nothing of consequence, really:

* Arbitrary capitalization in the intro's list (ESLint, custom elements).
* FWIW, this is my template: https://github.com/FND/javascript-boilerplate/blob/master/README.md